template/lib/utils.bash

```diff
@@ -59,7 +59,7 @@ install_version() {
 
   (
     mkdir -p "$install_path"
-    cp -r "$ASDF_DOWNLOAD_PATH/*" "$install_path"
+    cp -r "$ASDF_DOWNLOAD_PATH"/* "$install_path"
```

Comment:
can you describe this effect to me? is the quoted `/*` not interpretted correctly in your shell?

Reply:
> can you describe this effect to me? is the quoted `/*` not interpretted correctly in your shell?

In Bourne shell compatible shell (including bash), a quoted globing characters are passed to the invoked program literally. Pathname expansion is not performed on quoted words (both single and double quoted). 

Thus `cp -r "$ASDF_DOWNLOAD_PATH/*" "$install_path"` would only copy a file/directory with name `*` from `$ASDF_DOWNLOAD_PATH`.

On the other hand, a `"$ASDF_DOWNLOAD_PATH"/*` will be expanded correctly. The double quotes are still necessary to support `$ASDF_DOWNLOAD_PATH` referring to a path with spaces.

Reply:
Thanks for the clarification. I would have thought this would have come up before now seeing the number of people who cloned and used this template :shrug: